DirectProgramming/DPC++FPGA/Tutorials/Features/max_interleaving/src/max_interleaving.cpp

```diff
@@ -0,0 +1,202 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
```

Comment:
In your repo, line 7 is repeated on line 17
#include <CL/sycl.hpp>

Reply:
addressed